File: src/fibonacci.pas
program fibonacci;

function fib(n:integer): integer;
begin
    if (n <= 2) then
        fib := 1
    else
        fib := fib(n-1) + fib(n-2);
end;

var
    i:integer;

begin
    for i := 1 to 16 do
        write(fib(i), ', ');
    writeln('...');
end.

File: src/Fibonacci.go
package main

import "fmt"


func main() {
	fmt.Println(fib(7)) //Should print 13
}

func fib(n int) int{
	if n == 0{
		return 0
	}
	if(n <= 2){
		return 1
	}else{
		return fib(n-1)+fib(n-2)
	}
}
